cfy_manager/components/restservice/restservice.py

```diff
@@ -284,7 +284,7 @@ def _start_restservice():
 def _configure():
     _make_paths()
     _configure_restservice()
-    systemd.configure(RESTSERVICE, tmpfiles=True)
+    systemd.configure(RESTSERVICE)
```

Comment:
I think this might be the only tmpfiles usage in the whole code. If this is so, we might want to get rid of this option altogether

Reply:
Whoops, next commit :)